src/Athena/Controllers/AccountController.cs

```diff
@@ -1,183 +0,0 @@
-﻿using System;
-using System.Security.Claims;
-using System.Threading.Tasks;
-using Athena.Core.Models;
-using Athena.Core.Models.Identity;
-using Athena.Core.Repositories;
-using Athena.Models.Login;
-using Microsoft.AspNetCore.Authentication;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Identity;
-using Microsoft.AspNetCore.Mvc;
-using Serilog;
-
-namespace Athena.Controllers
```

Comment:
The account controller is a part of the login code that was added in #14. Please keep this controller.